### PR TITLE
Es6: common/modules/identity/validation-email.js

### PR DESCRIPTION
**Pull request by GHaberis**
## What does this change?

Convert common/modules/identity/validation-email.js to Es6.

## What is the value of this and can you measure success?

More Es6! :muscle:

## Does this affect other platforms - Amp, Apps, etc?

No

## Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

No

## Screenshots

N/A

## Tested in CODE?

No

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/identity/validation-email.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+// @flow
+
+import mediator from 'lib/mediator';
+import IdentityApi from 'common/modules/identity/api';
+
+const init = (): void => {
+    const resendButton = document.getElementsByClassName(
```
Comment (gustavpursche) [inline static/src/javascripts/projects/common/modules/identity/validation-email.js]: fastdom? 👀

File: static/src/javascripts/projects/common/modules/identity/validation-email.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        'js-id-send-validation-email'
+    )[0];
+
+    if (resendButton) {
+        resendButton.addEventListener('click', (event: Event): void => {
```
Comment (gustavpursche) [inline static/src/javascripts/projects/common/modules/identity/validation-email.js]: Thanks for killing bean!

**Review by gu-stav — APPROVED** [review]
Looks good to me. It would be nice to wrap the module into `fastdom.read` and `fastdom.write`, but it's an enhancement ...

File: static/src/javascripts/projects/common/modules/identity/validation-email.js (hunk 1 above)
Comment (GHaberis) [reply static/src/javascripts/projects/common/modules/identity/validation-email.js]: Ah yes, good suggestion, I should get into the habit of using fastdom.read/write when querying/writing to the DOM.

File: static/src/javascripts/projects/common/modules/identity/validation-email.js [hunk 3]
```diff
@@ -1,51 +1,72 @@
 // @flow
-
+import fastdom from 'fastdom';
```
Comment (gustavpursche) [inline static/src/javascripts/projects/common/modules/identity/validation-email.js]: You have to use

```js
import fastdom from 'lib/fastdom-promise';
```

**Review by SiAdcock — APPROVED** [review]
Great!

**Pull request merged**